source/infrastructure/tests/aws-instance-scheduler-remote-stack.test.ts

```diff
@@ -13,14 +13,16 @@
  *  permissions and limitations under the License.                            *
  *****************************************************************************/
 
- import { SynthUtils } from '@aws-cdk/assert';
- import '@aws-cdk/assert/jest';
+
  import {createSpokeStack} from './instance-scheduler-stack-factory';
+import {Template} from "aws-cdk-lib/assertions";
 
  /*
   * SnapShot Testing for the AwsInstanceSchedulerStack.
   */
  test('AwsInstanceSchedulerStack snapshot test', () => {
-   expect(SynthUtils.toCloudFormation(createSpokeStack())).toMatchSnapshot();
+   expect(Template.fromStack(createSpokeStack(),
+     {
+
```

Comment:
What's with the funky spacing?

Reply:
I think I forgot to delete the empty braces from when I tried a different fix. Should be fixed now though, thanks for noticing!